providers/tentacle.rb

```diff
@@ -94,7 +94,7 @@
     action :run
     cwd tentacle_install_location
     code <<-EOH
-      .\\Tentacle.exe new-certificate -e "#{cert_file}" --console
```

Comment:
This is actually on purpose.  The reason is that it will keep the same cert file if the tentacle instance is destroyed or removed.